Tests/PointFreeTests/__Snapshots__/EpisodeTests/testEpisodePage.1.Conn.txt

```diff
@@ -9,11 +9,11 @@
 
 ▿ Response
   Status 200 OK
-  Content-Length: 25436
+  Content-Length: 27455
```

Comment:
can see that the episode page increased by a whopping 2k! most of that is from an svg in the page now, and then a bit from the styleguide and design systems additions.